scripts/gerrit_jenkins/create_whitelist.py

```diff
@@ -0,0 +1,16 @@
+import sys
+import fileinput
+import json
+import re
+
+whitelist_file = sys.argv[2]
+with open(whitelist_file, 'w') as whitelist:
+  for line in fileinput.input():
+    r = re.compile("^\{.*\}")
```

Comment:
Please use raw text for regex `r"^\{.*\}"`
The re.compile part could be moved out from the for loop, so it is only compiled once.